### DISCUSSION TITLE
add some text to the issue template in yaml that is still visible after the issue is submitted

### DISCUSSION THREAD

**Discussion by RtYkk** [community/community#48397] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

It seems that there is no way to add some text to the issue template in yaml that is still visible after the issue is submitted.
[None of the current schema](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#checkboxes) can fit my requirements. Or I can just add them directly to the end of the template?

Thank you, guys.

**Comment by spenserblack** [comment, marked as answer]
What type of text did you want to add? The input labels should still be visible once the issue is submitted.

You can create a [template in Markdown](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-templates), which might be best to fit your needs currently.

Also, it's a bit of a hack, but, depending on what you want to do, you could use a GitHub Action to update the issue after it has been submitted.

**Reply by RtYkk** [reply]
I want to add some notes. Thank you. I will try it. The text area is much better. I will put my notes in the 'value' attributes.
